sos/report/plugins/cephadm.py

```diff
@@ -0,0 +1,55 @@
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.report.plugins import Plugin, RedHatPlugin, UbuntuPlugin
+
+
+class Cephadm(Plugin, RedHatPlugin, UbuntuPlugin):
+
+    short_desc = 'CEPH distributed storage managed via cephadm'
+
+    plugin_name = 'cephadm'
+    profiles = ('storage')
+
+    packages = (
+        'cephadm'
+    )
```

Comment:
These need to be tuples, so you'll need a trailing `,` even for 1-tuples.  E.g.

```python
profiles = ('storage',)
packages = ('cephadm',)
```

Reply:
`cephadm` is only available in very recent Ceph versions. Hence I was considering a separate plugin. Thanks for the comment on the tuples.